index.html

```diff
@@ -171,135 +171,135 @@
+<p><code>data-bind-show</code> adds the class <code>hide</code> to the node.  You can then use CSS like <code>.hide { display: none; }</code> or similar to hide the node and its children from view.</p>
 <div class="prepend-pre">
 <label>Enter a color:</label>
-<input type="text" bind="color">
-<p bind-show="color">
-  That color is <strong bind-show="color.length <= 4">not</strong> longer than 4 characters.
+<input type="text" data-bind="color">
+<p data-bind-show="color">
+  That color is <strong data-bind-show="color.length <= 4">not</strong> longer than 4 characters.
 </p>
 </div>
 
 <h2>Class bindings</h2>
-<p><code>bind-class</code> toggles a node's classes, given an object with class names for keys and booleans for values.</p>
+<p><code>data-bind-class</code> toggles a node's classes, given an object with class names for keys and booleans for values.</p>
 <div class="prepend-pre">
 <label>Enter a color:</label>
-<input type="text" bind="color">
-<h3 bind="color" bind-class="{blue: color == 'blue', green: color == 'green'}"></h3>
+<input type="text" data-bind="color">
+<h3 data-bind="color" data-bind-class="{blue: color == 'blue', green: color == 'green'}"></h3>
 </div>
 
 <h2>Checkboxes</h2>
 <p>The binding value of checkboxes is a boolean indicating the checked state.</p>
 <div class="prepend-pre">
-<input type="checkbox" bind="available">
-<span bind="available"></span>
+<input type="checkbox" data-bind="available">
+<span data-bind="available"></span>
 </div>
 
 <h2>Radio buttons</h2>
 <p>The binding value of radio buttons is the <code>value</code> attribute of the currently selected button. Make sure to use the same key for every button in the radio group.</p>
 <div class="prepend-pre">
-<p> State: <strong bind="state"></strong> </p>
-<input type="radio" name="state" bind="state" value="hidden" checked>
-<input type="radio" name="state" bind="state" value="published">
+<p> State: <strong data-bind="state"></strong> </p>
+<input type="radio" name="state" data-bind="state" value="hidden" checked>
+<input type="radio" name="state" data-bind="state" value="published">
 </div>
 
-<p>Alternatively, you may use <code>bind-checked</code>, which is a one-way binding of the <code>checked</code> attribute.</p>
+<p>Alternatively, you may use <code>data-bind-checked</code>, which is a one-way binding of the <code>checked</code> attribute.</p>
 
 <h2>Attribute bindings</h2>
 
 <p>The following DOM attributes can be bound to via:</p>
 
 <ul>
-  <li><code>bind-placeholder</code></li>
-  <li><code>bind-checked</code></li>
-  <li><code>bind-disabled</code></li>
-  <li><code>bind-href</code></li>
-  <li><code>bind-title</code></li>
-  <li><code>bind-readOnly</code></li>
+  <li><code>data-bind-placeholder</code></li>
+  <li><code>data-bind-checked</code></li>
+  <li><code>data-bind-disabled</code></li>
+  <li><code>data-bind-href</code></li>
+  <li><code>data-bind-title</code></li>
+  <li><code>data-bind-readOnly</code></li>
```

Comment:
Actually while you are at it, do you mind adding `data-bin-draggable`? Added it recently: https://github.com/Shopify/twine/commit/d89b6ed62c144fb58190ffbd43944708951eaf9c